packages/dx-react-core/docs/reference/getter.md

```diff
@@ -0,0 +1,13 @@
+# Getter Component Reference
+
+A React component that exposes a value that can be used or modified outside of the plugin containing it.
+
+## User reference
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+name | string | | A name used to identify a value.
+value | any | | A value.
+computed | ({ [getterName: string]: any }) => any | | A function used to calculate a value depending on values exposed by other getters. A value will be recomputed each time a dependency changes. Used when the `value` property is not defined.
```

Comment:
The second parameter of `computed` is `all actions`

Reply:
We will not publish full API for now.